v1.7.0/longhorn-engine/rockcraft.yaml

```diff
@@ -98,17 +98,6 @@ parts:
 
       ldconfig
 
-      # NOTE(aznashwan): pull and install liblonghorn
-      # https://github.com/longhorn/longhorn-engine/blob/v1.7.0/package/Dockerfile#L15-L22
-      LIBLONGHORN_COMMIT_ID="53d1c063b95efc8d949b095bd4bf04637230265f"
-      cd /usr/src
-      git clone https://github.com/rancher/liblonghorn.git
-      cd liblonghorn
-      git checkout ${LIBLONGHORN_COMMIT_ID}
-      make
-      make install
-
-  # NOTE(aznashwan): the Makefile targets are just the scripts found in the
```

Comment:
I guess this first line of the next paragraph should not be removed.

Reply:
Done